app/views/events/_form.html.erb

```diff
@@ -3,13 +3,7 @@
 <%= form_for(event) do |f| %>
   <% if event.errors.any? %>
     <div id="error_explanation">
-      <h2><%= pluralize(event.errors.count, "error") %> prohibited this event from being saved:</h2>
-
-      <ul>
-      <% event.errors.full_messages.each do |message| %>
-        <li><%= message %></li>
-      <% end %>
-      </ul>
+      <p style="text-align:center; color:red; font-weight:bold"><%= pluralize(event.errors.count, "error") %> prohibited this event from being saved:</p>
```

Comment:
Isn't this getting rid of the error messages? If that's what we want, it should at least merit a separate patch, with justification.